Mirror/Runtime/NetworkTime.cs

```diff
@@ -19,17 +28,27 @@ static double LocalTime()
             return span.TotalSeconds;
         }
 
-        public static void Reset(int windowSize)
+        public static void Reset()
         {
-            _rtt = new ExponentialMovingAverage(windowSize);
-            _offset = new ExponentialMovingAverage(windowSize);
+            _rtt = new ExponentialMovingAverage(PingWindowSize);
+            _offset = new ExponentialMovingAverage(PingWindowSize);
         }
 
         internal static NetworkPingMessage GetPing()
         {
             return new NetworkPingMessage(LocalTime()); 
         }
 
+        internal static void Update(NetworkClient networkClient)
```

Comment:
I just noticed that this is only for the client update.
it's still a bit magic, isn't it?
could assume that Update is on the server for one client, or similar.
Maybe use UpdateClient(NetworkClient client)?
Or is there a better way?

Reply:
Good call.   Renamed